modules/org.restlet.ext.apispark/src/org/restlet/ext/apispark/internal/conversion/swagger/v2_0/Swagger2Reader.java

```diff
@@ -549,32 +549,58 @@ private static void fillResponses(Operation swaggerOperation,
         }
 
         if (swaggerOperation.getResponses() != null) {
-            for (String key : swaggerOperation.getResponses().keySet()) {
-                Response swaggerResponse = swaggerOperation.getResponses().get(key);
-                org.restlet.ext.apispark.internal.model.Response response =
-                        new org.restlet.ext.apispark.internal.model.Response();
-
+            for (Entry<String, Response> responseEntry : swaggerOperation
+                    .getResponses().entrySet()) {
+                String key = responseEntry.getKey();
+                Response swaggerResponse = responseEntry.getValue();
                 int statusCode;
                 try {
                     statusCode = Integer.parseInt(key);
-                    response.setCode(statusCode);
-                } catch (Exception e) {
-                    // TODO: what to do with "Default" responses ?
-                    LOGGER.warning("Response " + key + " for operation " + swaggerOperation.getOperationId() +
-                            " could not be retrieved because its key is not a valid status code.");
+                } catch (NumberFormatException e) {
+                    if (!key.equals("default")) {
+                        LOGGER.warning(String.format(
+                                "Response '%s' for operation '%s' "
+                                        + "could not be retrieved because its "
+                                        + "key is not a valid status code.",
+                                        key, swaggerOperation.getOperationId()));
+                    }
                     continue;
                 }
+                fillResponse(statusCode, responseEntry.getValue(),
+                        swaggerOperation, operation, contract, parameters);
+            }
 
-                response.setDescription(swaggerResponse.getDescription());
-                response.setName(ConversionUtils.generateResponseName(statusCode));
-
-                fillOutputPayload(swaggerResponse, response, swaggerOperation, contract, parameters);
-
-                operation.getResponses().add(response);
+            Response defaultSwaggerResponse = swaggerOperation.getResponses().get(
+                    "default");
+            if (defaultSwaggerResponse != null
+                    && operation.getResponse(200) == null) {
```

Comment:
is it possible to have any other response than 200, such as 201 (resource created) or 204 (no entity)?
I mean, instead of checking operations with 200 status, is possible to check operations with a 2xx status (there a method on the org.restlet.data.Status class)?